docs/beyond_basics/101-173-contributing.rst

```diff
@@ -0,0 +1,36 @@
+
+* **Contributing on GitHub**: A most valuable contribution is your time. We are interested and grateful for opinions, bug reports, feature requests, patches, larger code contributions, or simply a notice what you use DataLad for. Find the relevant repository, be that `github.com/datalad/datalad <https://github.com/datalad/datalad>`_ (the main repository), `github.com/datalad-datasets <https://github.com/datalad-datasets>`_ (many open datasets), or any :term:`DataLad extension`, and open issues or pull requests. DataLad's `CONTRIBUTING <https://github.com/datalad/datalad/blob/master/CONTRIBUTING.md>`_ file has tons of technical and social information to get you started with code contributions. But don't be intimidated by the wealth of information you will find in there. We'll be happy to help you at any stage. Also, you can take a look at technical docs (`docs.datalad.org <http://docs.datalad.org/>`_) and in particular the :ref:`Design documents <design>` that shed light on the internal design principles of the software.
+
+* **Write an extension!** If you have unique use cases, you can write your own :term:`DataLad extension` for it, that can provide any number of additional DataLad commands that are automatically included in DataLad's command line and Python API. Our `extension template <https://github.com/datalad/datalad-extension-template>`_ is the best starting point. It contains an example command implementation, and will have test setup and packaging configurations in place already. If you want to, you can register your extension against DataLad's extension registry at `github.com/datalad/datalad-extensions <https://github.com/datalad/datalad-extensions>`_ -- if your project is included, we can continuously check whether current versions of DataLad work with your extension.
+
+* **Contributing to related projects** As open source software, we proudly stand on the shoulders of giants. The DataLad project wouldn't be possible without many other open source packages and projects. Helping them helps us, and you could do so in any of the ways described above, including documentation, tutorials, patches, support -- if you have a passion for `Haskell <https://www.haskell.org/>`_ or `C <https://en.wikipedia.org/wiki/C_(programming_language)>`_ you could even head over to `git-annex <http://source.git-annex.branchable.com/?p=source.git;a=summary>`_ or `Git <https://github.com/git/git>`_ themselves.
```

Comment:
Same as above, "Contributing to related projects" -> "Contribute to related projects"

Reply:
done